src/index.html

```diff
@@ -1,11 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>Hello webpack!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Leaderboard</h1>
+    </header>
+    <main class="container">
+      <section class="board">
+        <div class="board__head">
+          <h3 class="board__title">Recent Scores</h3>
+          <button type="button" class="board__refresh">Refresh</button>
+        </div>
+        <ul class="board__content">
+          <li class="board__item">Name: 100</li>
+          <li class="board__item">Name: 80</li>
+          <li class="board__item">Name: 75</li>
+          <li class="board__item">Name: 73</li>
+          <li class="board__item">Name: 70</li>
+        </ul>
+      </section>
+      <form action="#" class="form">
+        <h3 class="form__title">Add Your Score</h3>
+        <input
+          type="text"
+          class="form__name"
+          placeholder="Your Name"
+          required
+        />
+        <input
+          type="text"
+          class="form__score"
+          placeholder="Your Score"
+          required
+        />
```

Comment:
- [ ] As per the requirement in `index.html` you just have to create a root element and all the code will be added from `javascript`, it will make your page more dynamic and also make sense of using `javascript`. :slightly_smiling_face:

Reply:
We are not yet at that stage. These are placeholders to get the layout of the project.

Reply:
In the requirement it says `index.html` contain only root element.

![Screenshot from 2021-07-28 02-25-49](https://user-images.githubusercontent.com/22434660/127226185-98efe072-9d41-4e7a-a44a-93cc6b2726ff.png)

Reply:
@Dipeshtwis Does this root element speak to the ul tag which will dynamically load each player's score? If so I understand that and I will make the change so each li item is a clone that is appended. I do not have the requirement that you have sent.